### DISCUSSION TITLE
Feature request: Diagnostic error messages

### DISCUSSION THREAD

**Discussion by mobilemind** [community/community#3913] [Mobile]
How about useful error messages, error codes or a diagnostic log for the mobile app?

Actual example: 
```
Something happened.
Try again.
```

That current error message is quite useless, and leaves customers unable to problem solve or work toward a solution. Trying again doesn’t work. Who knows what happened?

**Comment by brianlovin** [comment]
Agreed. I've flagged this to our team, and added you +1 to our tracking issue. These error screens definitely aren't great 😅

**Comment by Vannsansan** [comment]
Hi
